js/controller.js

```diff
@@ -61,11 +84,30 @@
                     console.log(error);
                 });
 
-                $scope.greeting = config.greeting[Math.floor(Math.random() * config.greeting.length)];
+                if(typeof config.greeting.midday == 'undefined') {
+                    $scope.greeting = config.greeting[Math.floor(Math.random() * config.greeting.length)];
+                }
+
             };
 
             refreshMirrorData();
-            $interval(refreshMirrorData, 3600000);
+            $interval(refreshMirrorData, 1500000);
+
+            if(typeof config.greeting.midday != 'undefined') {
+                var greetingUpdater = function () {
+                    if (moment().hour() > 4 && moment().hour() < 11) {
```

Comment:
This feels a bit redundant (and very messy). I suggest you first determine what array of greetings to use and then setting them by random selection on the scope instead of having this complex if/else.

It should also be possible to only have 1 set of greetings if people don't feel like setting all of these up.